esp/esp/users/forms/user_profile.py

```diff
@@ -310,6 +310,14 @@ def clean(self):
 StudentInfoForm.base_fields['studentrep_expl'].widget.attrs['rows'] = 8
 StudentInfoForm.base_fields['studentrep_expl'].widget.attrs['cols'] = 45
 
+affiliationChoices = (
+    ('Pick', 'Pick one...'),
```

Comment:
This shouldn't be in choices, since it's not a valid choice; you can instead set `initial=` in the field, I think, and then django will do most of the work for you.